packages/components/src/dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,85 @@
+import { useState, ReactNode } from 'react';
+import React from 'react';
+import { type DropdownProps } from "@/dropdown/Dropdown.types";
+import { globalStyles } from "@/index.styled";
+import { DropdownContainer, DropdownLabel, DropdownChildren, DropdownItem, darkTheme } from "@/dropdown/Dropdown.styled"; 
+import chevronDown from "@/assets/chevron-down.svg"
+
+export const Dropdown = ({
+        label = "Select...", 
```

Comment:
'label' is a bit misleading, so you probably want to call it 'placeholder' instead